Python Basics/python_basics_home_task.py

```diff
@@ -0,0 +1,38 @@
+# Home task for Module 1 - Python Basics
+# Create list of 100 random numbers from 0 to 1000
```

Comment:
```suggestion
"""
Home task for Module 1 - Python Basics

Create a python script:
create list of 100 random numbers from 0 to 1000
sort list from min to max (without using sort())
calculate average for even and odd numbers
print both average result in console
"""
```
Let's agree to put a task full description to the very beginning on the file like this.